Makefile

```diff
@@ -13,6 +13,9 @@ build:
 	go build -o $(BUILD_ARCH)/$(BIN_DIR)/$(MAIN) cmd/$(MAIN)/main.go
 debug:
 	HUMAN_LOG=1 go run cmd/$(MAIN)/main.go
+acceptance:
+	SECRET_KEY=0C30662F-6CF6-43B0-A96A-954772267FF5 MONGODB_IMPORTS_DATABASE=test \
```

Comment:
Can we just add this key as a default in the config?

Reply:
done